Parse/src/main/java/com/parse/ParseFileController.java

```diff
@@ -41,7 +41,7 @@ public ParseFileController(ParseHttpClient restClient, File cachePath) {
   /* package */ ParseHttpClient awsClient() {
     synchronized (lock) {
       if (awsClient == null) {
-        awsClient = ParsePlugins.get().newHttpClient();
+        awsClient = ParsePlugins.get().awsClient();
```

Comment:
+1 for fileClient.

Let's use one for now until someone decides they need a separate one.  It was never exposed in the past.  I think the major reason is that there isn't a shared HttpClient because there is a parseClient() that is used for injecting Parse auth headers (see ParsePlugins.java)